app/controllers/katello/api/v2/organizations_controller.rb

```diff
@@ -168,11 +168,15 @@ def redhat_provider
 
     protected
 
+    def organization_export_content_permission?
+      org = Organization.friendly.find(params[:id])
+      org&.authorized?(:export_content) == true
```

Comment:
Nit: The comparison might be redundant.

Reply:
yes, unless this needs to always be a boolean

Reply:
updated :)